src/react/components/common/tagsInput/tagsInput.tsx

```diff
@@ -63,27 +67,31 @@ export const KeyCodes = {
  */
 const delimiters = [KeyCodes.comma, KeyCodes.enter];
 
+/**
+ * Component for creating, modifying and using tags
+ */
 export default class TagsInput extends React.Component<ITagsInputProps, ITagsInputState> {
```

Comment:
I feel like this component is getting overly complex and may be a good time to split up the functionality between a component that is used in the editor page vs. in the project settings page.  Let's discuss.

Reply:
Maybe `EditorTagsInput` that extends `TagsInput` and adds the click functionality?